packages/unlock-js/package.json

```diff
@@ -63,6 +63,7 @@
   "dependencies": {
     "@morpho-labs/ethers-multicall": "3.0.1",
     "@uniswap/v3-periphery": "1.4.3",
+    "axios": "1.2.1",
```

Comment:
Are we back to adding axios all over the place :( If so then let's open a ticket to. remove the _other_ fetch library?